docs/upgrade.html

```diff
@@ -1543,9 +1543,8 @@ <h5><a id="upgrade_1010_notable" href="#upgrade_1010_notable">Notable changes in
     <li> The new Java Consumer now supports heartbeating from a background thread. There is a new configuration
          <code>max.poll.interval.ms</code> which controls the maximum time between poll invocations before the consumer
          will proactively leave the group (5 minutes by default). The value of the configuration
-         <code>request.timeout.ms</code> must always be larger than <code>max.poll.interval.ms</code> because this is the maximum
-         time that a JoinGroup request can block on the server while the consumer is rebalancing, so we have changed its default
-         value to just above 5 minutes. Finally, the default value of <code>session.timeout.ms</code> has been adjusted down to
+         <code>request.timeout.ms</code> must always be smaller than <code>max.poll.interval.ms</code>, so we have changed
```

Comment:
It's better to still explain the reason why it's better to be larger. How about the following:

```
<code>request.timeout.ms</code> (default to 30 seconds) must always be smaller than <code>max.poll.interval.ms</code>, since that is the maximum time that a JoinGroup request can block on the server while the consumer is rebalance. Finally...
```

Reply:
It has been modified